src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -47,31 +48,60 @@ const CreateConceptForm = (props) => {
       <div className="concept-form-body">
         <div className="form-row">
           <div className="form-group col-md-7">
-            <label htmlFor="uuid"><h5>UUID</h5></label>
+            <label htmlFor="OpenMRS UUID"><h5>OpenMRS UUID (OCL External ID)</h5></label>
             <input
               type="text"
               className="form-control"
               readOnly={props.editable}
               onChange={props.handleChange}
-              value={props.state.id}
-              name="id"
+              name="external_id"
+              value={props.state.external_id}
               id="uuid"
               required
             />
             <small className="form-text text-muted">
             Only alphanumeric characters, hyphens, periods, and underscores are allowed.
             </small>
           </div>
+          {(!isEditConcept) && (
           <div className="form-group col-md-3 custom-field">
             <button
               type="submit"
               className="btn btn-sm btn-light"
               id="toggleUUID"
               onClick={props.toggleUUID}
             >
-            click here to manually enter
+                click here to manually enter
             </button>
           </div>
+          )}
+        </div>
+        <div className="form-row">
+          <div className="form-group col-md-7">
+            <label htmlFor="OCL ID"><h5>OCL ID</h5></label>
+            {(!isEditConcept) && (
+            <input
+              type="text"
+              className="form-control"
+              onChange={props.handleChange}
+              value={props.state.id}
+              name="id"
+              id="id"
+              required
+            />
+            )}
+            {(isEditConcept) && (
```

Comment:
Do we need a separate field for this in the code? I feel like we could have just added the `readonly=!isEditConcept` attribute to the previous input.